shell/main.sh

```diff
@@ -37,7 +37,7 @@ else
 fi
 
 if [ "$(gem search -i librarian-puppet)" = "false" ]; then
-  gem install librarian-puppet --no-rdoc --no-ri
+  gem install librarian-puppet --no-rdoc --no-ri -v 2.2.3
```

Comment:
Can I get a consult on pinning a dependency version in gem @nuclearsandwich? Is this safe/reasonable to do? I don't have much experience with it, and I don't know if you can have more than one version installed at a time (like cargo or npm) or if it pins for the whole system (like pip or homebrew).

Reply:
FWIW, I have no idea how gems or puppet work. But I was getting the error I referenced https://github.com/ros-infrastructure/roswiki_vagrant/issues/2#issuecomment-454508629 and I assumed gem was installing the most recent version (3.0.0) and this was the second most recent version and it gets farther when you do this.

Reply:
Given that I'm getting other errors further down the pipeline, I'd be curious to know what version of the library is currently installed on your VM, @wjwwood

Reply:
I haven't used this in years, it's not setup on any of my machines (virtual or otherwise).

Reply:
In this instance it's the reasonable thing to do unless/until we update or replace the actual puppet logic. It looks like the 3.0.0 release (Jan 2018) drops support for Ruby 1.x and Puppet 3.7 (There is a puppet 3.8 on Xenial that might work with it but that is a whole extra can of marionette strings).

Reply:
Sounds good, thanks for the input!